edx/blockstore
# Revert "Temporarily revert removal of Python 3.5 support"

## Description

This reverts the reversion of the Python 3.5 support.

## Author Comments, Concerns, and Open Questions

## Test Instructions

Refer to the original PR: https://github.com/edx/blockstore/pull/91

This PR should be placed on stage (be it via merge to be confirmed later, or via this branch directly) and the configuration directive should be added:

```
AWS_S3_SIGNATURE_VERSION = 's3v4' 
```

...in the relevant .yml file for stage. If this works as expected, production should be similarly updated as this change is rolled out there.
